Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -639,6 +639,26 @@ final class LayerTests: XCTestCase {
         XCTAssertEqual(output.shape, expected)
     }
 
+    func testReshapeGradient() {
+        let layer = Reshape<Float>(shape: [10, 2, 1])
+        let input = Tensor(shape: [1, 5, 4], scalars: (0..<20).map(Float.init))
+        let computedGradient = gradient(at: input, layer) { $1($0).sum() }
+        // The expected value of the gradient was computed using the following Python code:
+        // ```
+        //   Reshape = tf.keras.layers.Reshape(target_shape = (10, 2, 1))
```

Comment:
Use camel case for Python variables.
```suggestion
        //   reshape = tf.keras.layers.Reshape(target_shape = (10, 2, 1))
```